google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineStopTask.java

```diff
@@ -42,6 +44,9 @@ public AppEngineStopTask(AppEngineStop stop, String module, String version) {
 
   @Override
   public void execute(ProcessStartListener startListener) {
+    UsageTrackerProvider.getInstance()
```

Comment:
Well I think we know the environment implicitly since its only flexible environments which currently support stopping. Its true that some code is needed to know if its compat or not.